tools/docker_dev_setup.sh

```diff
@@ -29,8 +29,8 @@ dpkg-reconfigure --frontend noninteractive tzdata
 
 # Default values
 rocm_version="7.1.0"
-rocm_build_number="16623"
-rocm_job_name="compute-rocm-dkms-no-npi-hipclang"
+#rocm_build_number="16623"
```

Comment:
Let's not leave commented out code in here. Delete if it's not needed anymore. People can just pass the build number and job name in as parameters.